python/runtime/xgboost/predict.py

```diff
@@ -124,19 +121,8 @@ def predict_and_store_result(bst, dpred, feature_file_id, model_params,
     result_column_names.append(pred_label_name)
 
     line_no = 0
-    if is_pai:
-        driver = "pai_maxcompute"
-    else:
-        driver = conn.driver
-    with db.buffered_db_writer(driver,
-                               conn,
-                               result_table,
-                               result_column_names,
-                               100,
-                               hdfs_namenode_addr=hdfs_namenode_addr,
-                               hive_location=hive_location,
-                               hdfs_user=hdfs_user,
-                               hdfs_pass=hdfs_pass) as w:
+    with db.buffered_db_writer(conn, result_table, result_column_names,
+                               100) as w:
```

Comment:
Why remove the other parameters?

Reply:
Parameters like `hdfs_pass` can be passed by connection uri, they are driver specific. We have a unified way to save these params (in `conn.params dict`), we can get them  by `conn.param(param_name)` when needed.

Reply:
I see, the `db.py` file was folded and I missed the file.